solvebio/cli/main.py

```diff
@@ -158,26 +127,20 @@ class SolveArgumentParser(argparse.ArgumentParser):
                             'medium (<500M), large (>=500M)'
                 },
                 {
-                    'name': 'dataset_name',
-                    'help': 'The name of the dataset'
+                    'name': 'dataset_full_path',
+                    'help': 'The full path to the dataset in the following format: <domain>:<vault>:</object/path/variants>'  # noqa
                 }
             ]
         },
         'upload': {
             'func': data.upload,
             'help': 'Upload a file or directory to a SolveBio Vault',
             'arguments': [
-                {
-                    'flags': '--vault',
-                    'help': 'The name of the vault to use when '
-                            'creating a new dataset (via --create-dataset), '
-                            'defaults to your personal vault',
-                },
                 {
                     'flags': '--path',
```

Comment:
Maybe for consistency with other command line functions, this should also just be `--full-path`?

Reply:
im fine with that.